<?xml version="1.0" encoding="UTF-8"?>
<launch>
	<node pkg="arbotix_commander2" type="arbotix_commander2_node" name="commander2_joy" output="screen">
		<param name="serial_port" value="/dev/ttyUSB0" />
	</node>
</launch>